src/Symfony/Bridge/PhpUnit/bin/simple-phpunit.php

```diff
@@ -157,6 +157,11 @@
     ? ('#!/usr/bin/env php' === file_get_contents($COMPOSER, false, null, 0, 18) ? $PHP : '').' '.escapeshellarg($COMPOSER) // detect shell wrappers by looking at the shebang
     : 'composer';
 
+$prevCacheDir = getenv('COMPOSER_CACHE_DIR');
+if ($prevCacheDir && !(0 === strpos($prevCacheDir, '/') || ':' === substr($prevCacheDir, 1, 1) || 0 === strpos($prevCacheDir, '\\\\'))) {
+    $absoluteCacheDir = $oldPwd.'/'.$prevCacheDir;
```

Comment:
why not use realpath?

Reply:
Good catch. It also handle path like `~/.cache/`